src/Security/Member.php

```diff
@@ -1123,6 +1123,12 @@ public static function set_title_columns($columns, $sep = ' ')
 
     //------------------- HELPER METHODS -----------------------------------//
 
+    /* a method to allow using $LastName as an alternative to $Surname in templates since it is a commonly made mistake */
+    
+    public function getLastName() {
+      return $this->Surname;
```

Comment:
We have an automated "linter" that requires the code to have specific formatting, this is why the builds are failing for this pull request. The opening brace of this method (`{`) needs to be on a new line and the indentation is too little for the contents of the method. Here's the output from the linter - available by looking at the failing automated test run:

```
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 1128 | ERROR | [x] Opening brace should be on a new line
 1129 | ERROR | [x] Line indented incorrectly; expected at least 8
      |       |     spaces, found 6
```